src/GraphQL/Types/Scalars/EnumerationGraphType.cs

```diff
@@ -36,11 +36,9 @@ public EnumerationGraphType()
         /// <param name="deprecationReason">The reason this enumeration member has been deprecated; <see langword="null"/> if this member has not been deprecated.</param>
         public void AddValue(string name, string? description, object? value, string? deprecationReason = null)
```

Comment:
@Shane32 I find this method a bit problematic - I would move description as optional argument before deprecationReason.

Reply:
I agree; just be sure we have applicable migration documentation.

Reply:
done